lib/metadata/bucketfile/backend.js

```diff
@@ -432,7 +432,7 @@ class BucketFileInterface {
             const stream = db.createReadStream(requestParams);
             stream
                 .on('data', e => {
-                    if (extension.filter(e) === false) {
+                    if (extension.filter(e) < 0) {
```

Comment:
Same answer from https://github.com/scality/MetaData/pull/1009/files#r107927401: no, metadata only cares if the value is `< 0`, it does not need to understand the specific values (-1, -2, ...) unless we map these values to specific actions in metadata.